examples/pr_notifier/notifier.py

```diff
@@ -17,16 +18,15 @@
 S3_BUCKET = os.environ["S3_BUCKET"]
 SLACK_TOKEN = os.environ["SLACK_TOKEN"]
 SLACK_CHANNEL = os.environ["SLACK_CHANNEL"]
-SLACK_INSTANCE = os.getenv(
```

Comment:
This was unused and did not really make sense. It was to link to the Slack message instead of the PR.